### PR TITLE
Data Object Tree - Add object

### LINKED ISSUE #319: Data Object Tree - Add object
Part of https://github.com/pimcore/studio-ui-bundle/issues/261

Add new endpoint to add data object

### PR DESCRIPTION
**Pull request by lukmzig**
## Changes in this pull request
Resolves https://github.com/pimcore/studio-backend-bundle/issues/319

### CODE REVIEW

File: src/DataObject/Controller/AddController.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        description: 'Successfully added a new data object',
+        content: new IdJson('ID of created data object')
+    )]
+    #[IdParameter(type: ElementTypes::TYPE_DATA_OBJECT, name: 'parentId')]
+    #[AddDataObjectRequestBody]
+    #[DefaultResponses([
+        HttpResponseCodes::UNAUTHORIZED,
+        HttpResponseCodes::NOT_FOUND,
+    ])]
+    public function addAsset(
```
Comment (martineiber) [inline src/DataObject/Controller/AddController.php]:
```suggestion
    public function addDataObject(
```

**Review by martineiber — CHANGES_REQUESTED** [review]

File: src/DataObject/Controller/AddController.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+use Symfony\Component\Routing\Attribute\Route;
+use Symfony\Component\Security\Http\Attribute\IsGranted;
+use Symfony\Component\Serializer\SerializerInterface;
+
+/**
+ * @internal
+ */
+final class AddController extends AbstractApiController
+{
+    use PaginatedResponseTrait;
```
Comment (martineiber) [inline src/DataObject/Controller/AddController.php]: I think `getPaginatedCollection()` from this trait it not used? 
```suggestion
```

**Review by martineiber — CHANGES_REQUESTED** [review]

File: src/DataObject/Controller/AddController.php (hunk 2 above)
Comment (lukmzig) [reply src/DataObject/Controller/AddController.php]: Good catch! thank you :)

**Review by martineiber — APPROVED** [review]
:seal:

**Pull request merged**